backend/addcorpus/tests/test_corpus_views.py

```diff
@@ -16,23 +16,43 @@ def test_no_corpora(db, settings, admin_client):
     assert status.is_success(response.status_code)
     assert response.data == []
 
-def test_corpus_documentation_view(admin_client, basic_mock_corpus, settings):
-    response = admin_client.get(f'/api/corpus/documentation/{basic_mock_corpus}/')
+def test_corpus_documentation_list_view(admin_client, basic_mock_corpus, settings):
+    response = admin_client.get(f'/api/corpus/documentation/')
     assert response.status_code == 200
     pages = response.data
 
-    # check that the pages are sorted in canonical order
-    page_types = [page['type'] for page in pages]
+    # check that the pages specify canonical order
+    sorted_and_filtered = sorted(
+        (page for page in pages if page['corpus'] == basic_mock_corpus),
+        key=lambda page: page['index']
+    )
+    page_types = [page['type'] for page in sorted_and_filtered]
     assert page_types == ['General information', 'Citation', 'License']
 
     # should contain citation guidelines
-    citation_page = next(page for page in pages if page['type'] == 'Citation')
+    citation_page = next(
+        page for page in response.data if
+        page['type'] == 'Citation' and page['corpus'] == basic_mock_corpus
+    )
```

Comment:
Would `{'type': 'Citation', 'corpus': basic_mock_corpus} in response.data` suffice? For readability